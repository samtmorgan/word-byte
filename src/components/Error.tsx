import React from 'react';

export const Error = () => <div>ğŸ˜­ Ouch! Something went wrong, please try again.</div>;
