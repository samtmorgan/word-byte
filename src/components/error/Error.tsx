import React from 'react';

export default function ErrorPage() {
  return (
    <div className="page-container">
      <h1>ðŸ˜­ Ouch! Something went wrong, please try again.</h1>
    </div>
  );
}
