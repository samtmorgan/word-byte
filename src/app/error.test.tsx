import React from 'react';
import { render } from '@testing-library/react';
import Error from './error';

jest.mock('../components', () => ({
  ErrorPage: () => <div>mock error component</div>,
}));

describe('error page', () => {
  it('should return the error component', () => {
    const { container } = render(Error());
    expect(container).toMatchSnapshot();
  });
});
