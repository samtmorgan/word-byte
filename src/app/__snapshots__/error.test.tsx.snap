// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`error page should return the error component 1`] = `
<div>
  <div>
    mock error component
  </div>
</div>
`;
